src/components/Video/Video.js

```diff
@@ -0,0 +1,113 @@
+import { useEffect, useState } from "react";
+import ReactPlayer from "react-player";
+import { useParams } from "react-router";
+import { usePlaylistContext } from "../../contexts/playlist-context";
+import data from '../../data/videos'
+import PlaylistModal from "../PlaylistModal/PlaylistModal";
+import './Video.css'
+
+
+const playlists = [
+    {
+        title: "Demo",
+        videos: []
+    },
+    {
+        title: "Fav",
+        videos: []
+    },
+    {
+        title: "Wow",
+        videos: []
+    },
+    {
+        title: "Demo",
+        videos: []
+    },
+    {
+        title: "Fav",
+        videos: []
+    },
+    {
+        title: "Wow",
+        videos: []
+    },
+    {
+        title: "Demo",
+        videos: []
+    },
+    {
+        title: "Fav",
+        videos: []
+    },
+    {
+        title: "Wow",
+        videos: []
+    }
+]
+
+const Video = () => {
+    const [show, setShow] = useState(false);
+    const { videoId } = useParams();
+    const videoDetails = data.find((item) => item.id === videoId)
+    const {liked,bookmarked, userplaylists, addToLikedVideos, removeFromLikedVideos, removeFromBookmarkedVideos, addToBookmarkedVideos} = usePlaylistContext();
+    const isLiked = liked.videos.find((item)=> item.id === videoId)
+    const isBookmarked = bookmarked.videos.find((item)=> item.id === videoId)
+
+    const handleRemoveFromLikedVideos = () => {
+        removeFromLikedVideos(videoDetails);
+    }
+
+    const handleAddToLikedVideos = () => {
+        addToLikedVideos(videoDetails);
+    }
+
+    const handleRemoveFromBookmarkedVideos = () => {
+        removeFromBookmarkedVideos(videoDetails);
+    }
+
+    const handleAddToBookmarkedVideos = () => {
+        addToBookmarkedVideos(videoDetails);
+    }
+
+
+
+    return (
+        <div className="video-notes-container pt-1">
+          <div className="">
+                <div className="video-player">
+                    <ReactPlayer
+                        width="100%"
+                        height="100%"
+                        controls
+                        url={`https://www.youtube.com/watch?v=${videoDetails.url}`}
+                    />
+                </div>
+
+                <div className="video-details">
+                    <h1>{videoDetails.title}</h1>
+                    <div className="video-details-icons">
+                        {isLiked ?
+                            <i class="fas fa-thumbs-up" onClick={handleRemoveFromLikedVideos}></i>
+                            :
+                            <i className="far fa-thumbs-up" onClick={handleAddToLikedVideos}></i>
+                        }
+                        {isBookmarked ?
+                            <i class="fas fa-bookmark" onClick={handleRemoveFromBookmarkedVideos}></i>
+                            :
```

Comment:
You are using this video player code frequently so extract into a different component and use it